gateway-service-knoxtoken/src/main/java/org/apache/knox/gateway/service/knoxtoken/TokenResource.java

```diff
@@ -395,11 +397,22 @@ public Response doPost() {
   @GET
   @Path(GET_USER_TOKENS)
   @Produces({APPLICATION_JSON, APPLICATION_XML})
-  public Response getUserTokens(@QueryParam("userName") String userName) {
+  public Response getUserTokens(@QueryParam("userName") String userName, @QueryParam("mdName") String metadataName, @QueryParam("mdValue") String metadataValue) {
```

Comment:
Is it not possible to specify multiple metadata NV pairs in the query string? This seems to allow only a single pair.

Reply:
This is a good idea, let me check our possibilities here.

Reply:
Fixed.